packages/cli/src/editor/state/timeline.ts

```diff
@@ -0,0 +1,15 @@
+const localStorageKey = 'remotion.editor.timelineFlex';
+
+const persistTimelineFlex = (value = 0.2) => {
+	localStorage.setItem(localStorageKey, String(value));
+};
+
+const loadTimelineFlex = (): number => {
+	const item = localStorage.getItem(localStorageKey);
+
+	return item ? parseFloat(item) : 0.2;
+};
+
+export const useTimelineFlex = (): [number, (value?: number) => void] => {
+	return [loadTimelineFlex(), persistTimelineFlex];
```

Comment:
Since `ReflexElement` is not a [controlled component](https://reactjs.org/docs/forms.html#controlled-components), there's no need to use `useState` or something similar.